KabuKit/Classes/core/Swift/SceneSequence.swift

```diff
@@ -46,32 +53,102 @@ public class SceneSequence<ContextType, GuideType: Guide> : Scene, SceneContaine
         completion()
     }
     
+    public func contain<SceneType: Scene>(_ scene: SceneType) -> Bool {
```

Comment:
I think it's better to change `contain` to `contains` by following `Array#contains` .